source/io/module_eggShell.js

```diff
@@ -70,6 +70,15 @@
         publicAPI.eggShell = {};
 
         // Private Instance Variables (per vireo instance)
```

Comment:
This comment should go with variable `NULL`, `POINTER_SIZE` and `DOUBLE_SIZE`

Reply:
Oops. done.